pkg/transferer/transferer.go

```diff
@@ -57,7 +57,7 @@ func (t Transferer) readChunksFromSource(ctx context.Context, lc LoadStatusGette
 			switch lc.GetLatestStatus() {
 			case LoadStatusWait:
 				time.Sleep(MaxLoadWaitDuration)
-				log.Debug().Msgf("Got wait load status: putting chunks reading to sleep for %d seconds", MaxLoadWaitDuration)
+				log.Debug().Msgf("Got wait load status: putting chunks reading to sleep for %.0f seconds", MaxLoadWaitDuration.Seconds())
```

Comment:
better to just use `%v` (it will call `String()` method of `time.Duration` type)

example output: `2h`, `3s`, etc.

Reply:
Resolved in 662bee165546790b346d873a73a42c5da480f6cd